### DISCUSSION TITLE
Push notifications if device is locked

### DISCUSSION THREAD

**Discussion by paxter** [spacecowboy/Feeder#281] [Q&A]
I was looking for a new RSS feed reader with push notifications and landed here. Unfortunately, I don't getting notifications for new articles in configured feeds if the device is locked. Notifications are only working in unlocked device state.

Does this app support push notifications if device is locked?

For comparison, with [ReadYou](https://github.com/Ashinch/ReadYou) it's working fine.

**Comment by spacecowboy** [comment]
The app will notify for new articles if you have configured it to do so.

What I think is happening rather is that your device is not letting Feeder synchronize in the background (or you are just too impatiant).

* Make sure battery optimization is turned off for Feeder
* Set a reasonable sync frequency, like once per hour

Feeder is not designed (or even capable of) notifying you the moment something has been published somewhere.

**Reply by paxter** [reply]
> What I think is happening rather is that your device is not letting Feeder synchronize in the background (or you are just too impatiant).

Notifications are instantly pushed after unlocking the device. So that's for sure.

> Make sure battery optimization is turned off for Feeder

It's disabled. I'm using the same settings as for ReadYou.

> Set a reasonable sync frequency, like once per hour

15 minutes should be fine.

> Feeder is not designed (or even capable of) notifying you the moment something has been published somewhere.

I know how RSS works. Thanks.

I will stay with [ReadYou](https://github.com/Ashinch/ReadYou) as long notifications doesn't work in locked device state. I also noticed some rendering bugs on HTML parsing, but reporting bugs is obviously not wanted on this project.

Thanks for your fast reply anyway. :)

**Reply by spacecowboy** [reply]
Bug reports are quite welcome - via email. Just hit the button in the app.

If ReadYou works for you that's awesome.
